easybuild/easyblocks/n/netcdf.py

```diff
@@ -74,12 +77,12 @@ def configure_step(self):
                     self.cfg.update('configopts', '-D%s_INCLUDE_DIR=%s ' % (dep.upper(), incdir))
                     if dep == 'HDF5':
                         env.setvar('HDF5_ROOT', dep_root)
-                        libhdf5 = os.path.join(dep_root, dep_libdir, 'libhdf5.so')
+                        libhdf5 = os.path.join(dep_root, dep_libdir, 'libhdf5.'+SHLIB_EXT)
```

Comment:
also, I personally have a preference for this:

```python
'libhdf5_hl.%s' % shlib_ext
```

but that's less of an issue w.r.t. code style